bin/dry-run.rb

```diff
@@ -39,11 +39,10 @@
 require "etc"
 unless Etc.getpwuid(Process.uid).name == "dependabot"
   puts <<~INFO
-    bin/dry-run.rb is only supported in a development container.
+    bin/dry-run.rb is best supported in a development container.
 
-    Please use bin/docker-dev-shell first.
+    Consider using bin/docker-dev-shell first.
```

Comment:
```suggestion
    bin/dry-run.rb is only supported in a development container.
    
    Please use bin/docker-dev-shell first unless you have a specific reason not to.
```

Reply:
May I suggest an ENV instead that communicates the same thing but also allows me to run this without warnings? Something like `FORCE_UNSUPPORTED_USE_OF_DRY_RUN_WITHOUT_DOCKER_DEV_SHELL`?

Reply:
So, keep the current wording and keep this a hard error, but skip it if this ENV is set?

Reply:
Yeah, that's probably a better move so that someone has to explicitly opt into bypassing the `exit`.

An env var or a CLI arg would be fine too... I'm not too picky, whatever is smoothest for your workflow... Although a shorter env var name might be better 😆 Maybe something like `ALLOW_DRY_RUN_STANDALONE`? Or a cli arg of `--allow-standalone`... up to you.

Reply:
For my workflow ENV var works best, because it's silent and I can set it permanently 👍.

Now onto naming. My verbose proposal was precisely to accommodate your suggestion of not giving the impression that this running mode is supported in any way. But if you think `ALLOW_DRY_RUN_STANDALONE` is fine, that works for me!

I think a flag does give the impression of support so I would avoid that in favor of an ENV variable.

Reply:
I updated it to use `ALLOW_DRY_RUN_STANDALONE`.